fetchhub/assetlist.json

```diff
@@ -28,7 +28,7 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/fetchhub/images/fet.png",
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/fetchhub/images/fet.svg",
```

Comment:
Can we make sure the svg keeps a square canvas? Currently is rectangular.

Reply:
done